<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设定进价与处方价</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/back.jpg);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>药品名称：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>药品进价：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="UnitLine"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>药品处方价：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="PPLine"/>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="comfirmButton">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="background_image.qrc"/>
 </resources>
 <connections/>
</ui>
